pkg/crc/machine/status.go

```diff
@@ -44,11 +44,16 @@ func (client *client) Status() (*types.ClusterStatusResult, error) {
 	}
 
 	if vmStatus != libmachinestate.Running {
-		return &types.ClusterStatusResult{
-			CrcStatus:        state.FromMachine(vmStatus),
-			OpenshiftStatus:  types.OpenshiftStopped,
-			OpenshiftVersion: crcBundleMetadata.GetOpenshiftVersion(),
-		}, nil
+		clusterStatusResult := &types.ClusterStatusResult{
+			CrcStatus: state.FromMachine(vmStatus),
+		}
+		if crcBundleMetadata.GetBundleType() == bundle.OpenShift {
+			clusterStatusResult.OpenshiftStatus = types.OpenshiftStopped
+			clusterStatusResult.OpenshiftVersion = crcBundleMetadata.GetOpenshiftVersion()
+		} else {
+			clusterStatusResult.PodmanVersion = crcBundleMetadata.GetPodmanVersion()
+		}
```

Comment:
If we add more bundle types, this will quickly become messy, I've tried to address this differently in https://github.com/cfergeau/crc/commits/machine-refactor2